subworkflows/local/fastq_to_cram/main.nf

```diff
@@ -78,26 +82,30 @@ workflow FASTQ_TO_CRAM {
                 files
             ]
         }
-        .groupTuple(by:[0])
-        .dump(tag: "FASTQ_TO_CRAM: bam per sample",pretty: true)
-        .map { meta, files ->
+        | groupTuple(by:[0])
+        | map { meta, files ->
             return [meta, files.flatten()]
         }
-        .set{ch_bam_per_sample}
+        | multiMap { meta, bam ->
+            bam:   [meta, bam]
+            fasta: [meta, WorkflowMain.getGenomeAttribute(meta.genome, 'fasta')]
```

Comment:
Also these file won't get validated by nf-validation, although that shouldn't really hurt if it uses the genomes/igenomes configs

Reply:
Suggestions to have the paths validated are welcome!

Reply:
The only real solution would be to define which ones you need before validation and set them to the right parameters.  I'll see if I can find a better solution